package/gluon-check-connection/check_site.lua

```diff
@@ -0,0 +1,3 @@
+if need_table(in_domain({'check_connection'}), nil, false) then
+	need_string_array_match(in_domain({'check_connection', 'targets'}), '^[%x:]+$')
```

Comment:
I don't think `in_domain` necessarily makes sense here. You could also define a connection check target in the internet, in which case a single setting could be used for a while site.